Project_1/Styles.css

```diff
@@ -1,76 +1,150 @@
+TEXT STYLES 
+==============*/
+
+header {
+    border-bottom: 10px solid #018a80;
+    border-top: 10px solid #018a80;
+    margin: 50px 0 100px;
+}
+
 h1 {
-    font-size: 100px;
-    color: #d07f7d;
-    margin-right: 300px;
-    margin-left: 650px;
-    line-height: 1.3;
+    color: #a72323;
+    font-family: 'Noto Sans', sans-serif;
+    font-size: 95px;
+    margin: 10px 300px 50px 650px;
 }
 
-h2 {
-    font-size: 25px;
+h2,
+h3,
+h4 {
     color: #808080;
-    margin-right: 300px;
-    margin-left: 650px;
-    opacity: 70%;
+    font-family: 'Noto Sans', sans-serif;
     font-style: normal;
     font-weight: normal;
 }
 
-h3 {
-    margin-right: 300px;
-    margin-left: 650px;
-    color: #808080;
-    font-size: 16px;
-    font-weight: normal;
+h2 {
+    font-size: 25px;
 }
 
+h3,
 h4 {
-    margin-right: 300px;
-    margin-left: 650px;
     font-size: 16px;
-    color: #808080;
-    font-weight: normal;
+}
+
+h2,
+h3,
+h4 {
+    margin: 20px 300px 20px 650px;
 }
 
 p {
+    font-family: 'Merriweather', serif;
     font-size: 18px;
-    margin-right: 300px;
-    margin-left: 650px;
     line-height: 2.2;
+    margin: 30px 300px 30px 650px;
 }
 
-/*i am not sure how to edit either image individually, so right now the Author photo is very much warped /*
-/*img {
-    margin-right: 400px;
-    margin-left: 750px;
-    max-width: 100%;
-    height: 200px;
-    width: 50%;
-    src: "Carl_Zimmer.jpg";/*
+figcaption {
+    background-color: whitesmoke;
+    border: 3px solid whitesmoke;
+    font-size: 13px;
+    margin-top: -50px;
+    position: absolute;
+}
+
+/*===========
+IMAGES
+=============*/
+
+.octopus-image {
+    margin: 70px 200px 50px 500px;
+    width: 110%;
+}
+
+.author-photo {
+    background-color: #018a80;
+    border: 10px solid #018a80;
+    margin: 150px 350px 100px 650px;
+    width: 9%;
+}
+
+/* ============
+CLASSES
+===============*/
+
+.quote2,
+.quote1 {
+    color: #018a80;
+    font-size: 50px;
+    font-weight: 700;
+    margin: -30px 0 -60px 600px;
 }
 
-hr {
-    width: 300px;
-    margin-left: auto;
-    margin-right: auto;
-    height: 100px;
-    background-color: #666;
-    opacity: 0.5;
+/*pullquote*/
 
+.quote2,
+.quote1,
+.author-description {
+    font-family: 'Merriweather', serif;
 }
 
+.pullquote p {
+    background-color: #018a80;
+    border: 10px #018a80;
+    color: #ffffff;
+    font-size: 20px;
+    margin: 10px 500px 10px;
+    padding: 20px;
+}
 
+.pullquote p,
+.author_header {
+    font-family: 'Noto Sans', sans-serif;
+}
 
-/*comments look like this*/
+/*about the author*/
+
+.author-header {
+    font-size: 25px;
+    margin: -360px 300px 0 850px;
+    text-transform: uppercase;
+}
+
+.author-description {
+    font-size: 15px;
+    margin: -300px 300px 0 850px;
+}
+
+.author-header,
+.author-description {
+    color: #808080;
+    position: absolute;
+}
+
+/*footer*/
+
+.footer {
+    background-color: #a72323;
+    height: 500px;
+    margin-top: 100px;
+    width: 100%;
```

Comment:
A `div` by default will fill the width of its parent, so you can safely delete this.